### PR TITLE
[KR] 07-2.md

### PR DESCRIPTION
**Pull request by marofmar**
Korean translation 07-2.md file, please review the file. 
I have also modified _config.yml to list 07-2.md on the page. 
As I found 07.md file has not been translated to Korean yet, I plugged English 07.md file in the place for the Korean one temporarily.

### CODE REVIEW

**Comment by Atcold** [comment]
Thank you! @skyil7, can you review this PR plz?

File: docs/ko/week07/07-2.md [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+1. Given a point $y_1$, can we predict $y_2$?
+2. Given $y$, can we find the closest point on the data manifold? -->
+
+에너지 함수에 대해 알게되면, 다음과 같은 질문에 대해 답을 할 수 있게된다:
+1. $y_1$ 의 점이 주어졌을 때, $y_2$ 를 예측해 낼 수 있는가?
+2. $y$ 가 주어졌을 때, 이 데이터 매니폴드<sup>manifold</sup> 와 가장 가까운 점을 찾아낼 수 있는가? 
+
+<!-- K-means belongs to architectural methods (as opposed to contrastive methods). Hence we do not push up the energy anywhere, all we do is push the energy down in certain regions. One disadvantage is that once the value of $k$ has been decided, there can only be $k$ points that have $0$ energy, and every other point will have higher energy that grows quadratically as we move away from them. -->
+
+K-means 는  (대조적 방법과 반대인) 건축적 방법에 속한다. 따라서 우리는 에너지를 아무데나 밀어 올리지 않고, 특정 지역의 에너지를 낮춘다. 한 가지 단점은 $k$ 의 값이 결정되면, 에너지 값이 $0$ 인 데이터 포인트가 단 $k$ 로 한정된다는 것과, 다른 모든 포인트의 에너지 값은 그 데이터 포인트에서 멀어질수록 이차적으로<sup>quadratically</sup> 증가하는 높은 값을 갖는다는 점이다.
```
Comment (skyil7) [inline docs/ko/week07/07-2.md:237]: architectural 의 번역은 `건축적`보단 `구조적`을 써주세요!

File: docs/ko/week07/07-2.md [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+
+K-means 는  (대조적 방법과 반대인) 건축적 방법에 속한다. 따라서 우리는 에너지를 아무데나 밀어 올리지 않고, 특정 지역의 에너지를 낮춘다. 한 가지 단점은 $k$ 의 값이 결정되면, 에너지 값이 $0$ 인 데이터 포인트가 단 $k$ 로 한정된다는 것과, 다른 모든 포인트의 에너지 값은 그 데이터 포인트에서 멀어질수록 이차적으로<sup>quadratically</sup> 증가하는 높은 값을 갖는다는 점이다.  
+
+<!-- ## Contrastive Methods -->
+## 대조적 방법
+
+<!-- According to Dr Yann LeCun, everyone will be using architectural methods at some point, but at this moment, it is contrastive methods that work for images. Consider the figure below which shows us some data points and contours of the energy surface. Ideally, we want the energy surface to have the lowest energy on the data manifold. Hence what we would like to do is lower the energy (i.e. the value of F(x,y))around the training example, but this alone may not be enough. Hence we also raise it for the $y$'s in the region that should have high energy but has low energy. -->
+
+얀 르쿤 박사에 따르면 모든 사람이 언젠가는 컨축적 방법을 사용할 것이지만, 현 시점에서는 이미지 데이터에 대해 잘 동작하는 것은 대조적 방법이다. 에너지 표면의 윤곽<sup>contours</sup>과 데이터 포인트들을 보여주는 아래 그림을 참고하자. 이상적으로, 우리는 에너지 표현이 데이터 매니폴드 위에서 최소의 에너지를 갖기를 바란다. 따라서 에너지 훈련 예제 주변의 에너지 값 (즉, F(x,y)의 값) 을 낮추려고 하지만 , 이것 만으로는 충분하지 않을 수 있다. 따라서 높은 에너지값을 가져야 하는데 낮은 에너지값을 가지고 있는 부분의 $y$ 값을 올린다.
```
Comment (skyil7) [inline docs/ko/week07/07-2.md:244]: `컨축적` 오타 있습니다. 이 부분도 구조적으로 바꾸는 것이 좋을 것 같습니다!

File: docs/ko/week07/07-2.md [hunk 3]
```diff
@@ -0,0 +1,349 @@
...
+
+<!-- There are several ways to find these candidates $y$'s that we want to raise energy for. Some examples are:
+1. Denoising Autoencoder
+2. Contrastive Divergence
+3. Monte Carlo
+4. Markov Chain Monte Carlo
+5. Hamiltonian Monte Carlo -->
+
+우리가 에너지 값을 높이고자 하는 $y$ 후보들을 찾는 몇 가지 방법이 있다. 이 방법의 몇 가지 예시는 다음과 같다:
+1. Denoising Autoencoder<sup>디노이징 오토엔코더</sup>
```
Comment (skyil7) [inline docs/ko/week07/07-2.md:265]: Encoder는 인코더로 쓰는 경우가 많습니다!

**Comment by marofmar** [comment]
[KR] 07-2.md 수정 완료했습니다!
06-2는 따로 수정한것 없는데, 제가 뭘 눌렀나봅니다, 무시해주세요! 

Done editing 07-2.md based on review comments! Thank you!
And for the 06-2.md raising conflicts, please ignore them nothing has been modified, maybe I clicked sth while checking the format

**Comment by skyil7** [comment]
수고하셨습니다! 적용했습니다.

**Comment by Atcold** [comment]
Yay! Thank you both!

P.S. Nice mask, @skyil7!

**Pull request merged**